### ISSUE TITLE
USA Aurora cannot be destroyed when it locks on target

### ISSUE THREAD

**Issue by xezon** [TheSuperHackers/GeneralsGamePatch#655]
USA Aurora cannot be destroyed when it locks on target. Aurora is banned in semi player games because of it. There is no counter to it, other than attacking the airfield in the enemy base. Maybe Aurora should not have this feature.

### Proposal 1
Aurora is not invincible, but stealth while attacking. This way stealth detection + anti air can kill it. Similar to Stealth Fighter. Nerf for base attacks, but buff for army attacks in the open, because the opponent will not see attacks coming.

### Proposal 2
Aurora is not invincible, but tanks a lot more damage while attacking. This way it can still be shot down if there is enough firepower present. Nerf for base + army attacks.

### Proposal 3
Aurora cost is changed.

### Proposal 4
Increase reload time. Then other players have more time to kill it while in the airfield and have more time to recover in between Aurora attacks.

### Proposal 5
Remove SF GP unlock, and give it to the Aurora instead

### Proposal 6
Increase the turn radius of the Aurora's slow locomotor so it has a higher chance of getting shot down after it attacks

### Proposal 7
Lessen its invincibility miss distance so AA weapons re-target faster after the Aurora comes out of supersonic mode and weapons with large AoE (nuke mig missiles) have a chance of killing them even in supersonic mode

### Proposal 8
Alpha Aurora only: Increase delay of their FAB explosion to make it less reliable in hitting moving targets

### Proposal 9
Decrease Aurora HP or increase the damage it takes while in low fuel

### Proposal 10
Aurora is not invincible for Avengers and Laser Turrets (Light Speed > Aurora Speed)

**Comment by MTKing4** [comment]
Proposal 2.

**Comment by ExiLe-ZH** [comment]
Don’t see reason to remove this feature tbf, it’s the only reason to go for them (and the Alpha because of it’s damage ofcourse). Other than that non-alphas are hardly ever economically viable. They can’t destroy more than they’re worth in 1 run and there’s no guarantee they survive when killing high value targets, like SW’s. 99% of the time it’s a 10k vs 5k trade (+airfield that takes awful lot of space). Sure they can be used to kill defenses at frontlines but then it’s still a risky overinvestment. 

I forsee that with either proposal noobs will still ban them for being OP (while they never were). 

Noobs will always play with weird rules anyway and you can’t balance around that, like the No-Rush rule is exactly the same as the No-Whatever rule.

Also, there are more OP units in the game like DB’s but ppl here like to BUFF them better with detonation buttons? This doesn’t make much sense really.

Another fun fact: EMP patriot misseles sometimes hit home and take out Auroras midflight.

Btw, I’m not trying to shut down any conversation here and I like to brainstorm along in attempt to put Auroras in a better spot but I think they already are in the best spot possible (after price swap of 1.04+ that makes Alpha 2.5k and Au 2k).

**Comment by MTKing4** [comment]
Aurora/alpha Units are the worst designed zh units by far, it's an unfair Uncounterable Unit for both Noobs and pros alike, they can hunt a player with dozen CCs and Dozers and nothing the enemy can do about that, does that make sense to you?

if you're worried about them not hitting more than their worth you can just reduce their cost and problem solved ezpz

**Comment by xezon** [comment]
Making them take a little damage while locked sounds like a reasonable change to me. This way it can't penetrate everything. Given its supersonic speed it will still get through rockets and defenses quickly, so it should not change anything in regular Pro Games. We can design it so it only affects Bunker Game, where one would expect nothing can get through his massive defensive line after investing a lot of firepower into it. I think that is the main issue.

Noobs want their fortress to hold and mean something.

**Comment by MTKing4** [comment]
An example of a pro Game Affected Directly by this Feature

+ Inf/tank having the whole map yet loose to a camping Superweapon behind only EMP and 3-4 alpha Auroras dozer hunting the enemy & killing migs (Very common)
+ GLAs Vs Airforce or any USA LateGame Campfest killing enemy superweapons with PDL Auroras or normal ones passing a hundred stinger site and quad (Less common)

**Comment by ExiLe-ZH** [comment]
> Aurora/alpha Units are the worst designed zh units by far, it's an unfair Uncounterable Unit for both Noobs and pros alike, they can hunt a player with dozen CCs and Dozers and nothing the enemy can do about that, does that make sense to you?
> 
> if you're worried about them not hitting more than their worth you can just reduce their cost and problem solved ezpz

It’s a different story with Alpha’s for sure, but I’ve never seen a normal Aurora change a game in the 30k+ games I’ve played (neither in high or low level games). Hunting doesn’t even work when the dozer is moved with good timing (no skills required, just need to pay attention).

Nothing about the core-design needs to be  changed as there’s nothing wrong with it, Vanilla Aurora proves this fact. I think it’s very cool concept that makes the unit unique.

Now if we talk about Alpha’s, there’s only 1 factor that makes them OP, that’s the damage (+radius). Should that be nerfed? Well maybe ever so slightly, we can talk about it (1.06 levels are a joke though). Should they be more expensive than 2.5k? Imo not. 

SW is supposed to be insanely strong lategame and overall the faction is pretty well balanced. Noobs letting SW build up is well… a noob problem. Sure, some army combo’s in teamgames make it pretty impossible to rush SW early on but yea, that’s ZH’s rock/paper/scissors army system for you (should try and team his mate, while SW is building up).

**Comment by ExiLe-ZH** [comment]
> Making them take a little damage while locked sounds like a reasonable change to me. This way it can't penetrate everything. Given its supersonic speed it will still get through rockets and defenses quickly, so it should not change anything in regular Pro Games. We can design it so it only affects Bunker Game, where one would expect nothing can get through his massive defensive line after investing a lot of firepower into it. I think that is the main issue.
> 
> Noobs want their fortress to hold and mean something.

Still nothing wrong with them in uber bunker games, what can they really kill? (again, talking about Vanilla Aurora's). If the idea is to take out a SW I rather have 2 Particles myself, or 4 SDZ.

Like the only good TRADE they can make is killing an USA SDZ (or maybe an oil or something) but even then, I rather have a SDZ myself. Oh and they give away awful lot of XP aswell.

**Comment by MTKing4** [comment]
> It’s a different story with Alpha’s for sure, but I’ve never seen a normal Aurora change a game in the 30k+ games I’ve played (neither in high or low level games). Hunting doesn’t even work when the dozer is moved with good timing (no skills required, just need to pay attention).

I've given an example of when Normal auroras being very frustrating to deal with, wether they're game changers or not does not dictate that there's nothing wrong with them. A lot of things are badly designed yet they don't change the outcome of a game by themselves, but simply make the experience worse, like having minigunners hard to crush for example.

And if you want to justify its design when it comes to hunting based on skill, or paying attention (which is basically skill since you need it to be able to pay attention everywhere) then this can be easily disregarded when an equally skilled player times the bomb just in front of the moving dozer. Basing your logic on skill is very wrong in that matter as far as i can see

> Now if we talk about Alpha’s, there’s only 1 factor that makes them OP, that’s the damage (+radius)

When it comes to kicking or hunting players with a just a few clicks, no. That's not what's making them OP.

> SW is supposed to be insanely strong lategame and overall the faction is pretty well balance

Nothing is supposed to be insanely strong at any point if you want to have a balanced game

the descriptions Insanely strong and balanced contradict one another

**Comment by ExiLe-ZH** [comment]
> > It’s a different story with Alpha’s for sure, but I’ve never seen a normal Aurora change a game in the 30k+ games I’ve played (neither in high or low level games). Hunting doesn’t even work when the dozer is moved with good timing (no skills required, just need to pay attention).
> 
> I've given an example of when Normal auroras being very frustrating to deal with, wether they're game changers or not does not dictate that there's nothing wrong with them. A lot of things are badly designed yet they don't change the outcome of a game by themselves, but simply make the experience worse, like having minigunners hard to crush for example.
> 
> And if you want to justify its design when it comes to hunting based on skill, or paying attention (which is basically skill since you need it to be able to pay attention everywhere) then this can be easily disregarded when an equally skilled player times the bomb just in front of the moving dozer. Basing your logic on skill is very wrong in that matter as far as i can see
> 
> > Now if we talk about Alpha’s, there’s only 1 factor that makes them OP, that’s the damage (+radius)
> 
> When it comes to kicking or hunting players with a just a few clicks, no. That's not what's making them OP.
> 
> > SW is supposed to be insanely strong lategame and overall the faction is pretty well balance
> 
> Nothing is supposed to be insanely strong at any point if you want to have a balanced game
> 
> the descriptions Insanely strong and balanced contradict one another

Ofcourse whether units or abilities are gamechanging dictates whether it should be buffed or nerfed. You using the word OP already says it's supposedly game changing (impacting balance), because if it doesn't, then it can't be OP either.

Seems you're completely disregarding anything I've said about trading (basic balance concept) and it's really beyond me how you think Vanilla Aurora's are OP. What is the frustrating part? Them taking out your SW or what else? I gladly take any of those trades haha. I would also applaud a player who specifically invests in aurora's and manages to hunt me, would call it classy even (even more so if he force fires my moving dozer). 

Come on really, how many times you've hunted by a Vanilla Aurora (since it's a so easy)? And how many times by King Raptors, Techs or Vees etc.?

Not every ability at any timeframe has to be counterable, that's not what makes up balance, what makes up balance are the pros and cons in total. ZH has endless examples of uncounterable abilities and genpowers. GLA teleporting entire armies from 1 tunnel to another is frustrating too in your opinion and should be removed/nerfed? Other C&C's have units that can teleport at any time, that bad design aswell?

MG example is not good, not anywhere is it hinted or described that it's smaller hitbox is a design feature / pro ability. This should be fixed. Aurora passing defenses and bombing stuff deep in one's base is literally what it's meant to do and it's described very clearly.

Also not sure what you're getting at regarding SW and early/late game balance, winrate data proves it's not an overpowered army. Let noobs play with rules if that balances out their gameplay (in their minds), but again, let's not balance around this nonsense.

**Comment by xezon** [comment]
From Noob and Semi player perspective Aurora is OP because it cannot be killed when it attacks. Invested money does not matter in camp game. Noobs don't care about that at all. If Aurora can fly through defense 100% it is frustrating for victim. Aurora can kill power plants and oils with 100% chance. USA does not need such an OP ability of **unlimited 100% success attacks**.

**Comment by MTKing4** [comment]
> You using the word OP already says it's supposedly game changing (impacting balance), because if it doesn't, then it can't be OP either.

I never said vanilla auroras were OP, at least not in the pro scene, i only said that they are badly designed units that can be frustrating to deal with in certain situations, if anything I'm pushing for them to be nerfed by reducing their cost to 1.8K or even lower while removing the nonsensical feature that is supersonic invincibility.

> Not every ability at any timeframe has to be counterable, that's not what makes up balance, what makes up balance are the pros and cons in total

That's not how leading game industries balance their games, you need to go see how imbalanced features are being treated in other RTS games, you'd be surprised seeing that you think that way, and the reception is always positive from the majority of their playerbase, because it makes sense. I've played almost every RTS game that comes out once it comes out, trust me, i know this is the right approach.

> GLA teleporting entire armies from 1 tunnel to another is frustrating too in your opinion and should be removed/nerfed? Other C&C's have units that can teleport at any time, that bad design aswell?

No. tunnel can be countered in almost a hundred different ways,
Now imagine a tunnel that will pop no matter you do to it, that's how you could compare it to an aurora

**Comment by ZekeDlyoung** [comment]
I don't think the Aurora invulnerability should be touched, as it is a unique defining characteristic of it.

That said I have been thinking of how to nerf it in other ways:

1. Increase reload time, so other players have more time to kill it while in the airfield/ have more time to recover in between Aurora attacks
2. Remove SF GP unlock, and give it to the Aurora instead (I don't think build cost needs to be further increased they're already very expensive)
3. Increase the turn radius of the Aurora's slow locomotor so it has a higher chance of getting shot down after it attacks
4. Lessen its invincibility miss distance so AA weapons re-target faster after the Aurora comes out of supersonic mode and weapons with large AoE (nuke mig missiles) have a chance of killing them even in supersonic mode
5. Alpha Aurora only: Increase delay of their FAB explosion to make it less reliable in hitting moving targets
6. Decrease Aurora HP or increase the damage it takes while in low fuel
7. All of the above

If you are set on removing its super-sonic ability, a more interesting alternative to giving it tons of HP is to re-use the lock-on mechanic from ZH. Instead of being invincible while attacking, AA weapons simple have a delay to adjust their aim before firing

**Comment by statesman01** [comment]
Here's mine. Disable SUPERSONIC evasion, increase volley evasion instead. For at least if it is overwhelmed, surely this plane should be shut down.

**Comment by ZekeDlyoung** [comment]
> Here's mine. Disable SUPERSONIC evasion, increase volley evasion instead. For at least if it is overwhelmed, surely this plane should be shut down.

What do you mean by volley evasion?

**Comment by phantomatlas123** [comment]
> 
Its the countermeasure.  It throws volleys it can be modified how many volley to be fired. Aurora would be vulnerable if overwhelmed.

**Comment by phantomatlas123** [comment]
> > Here's mine. Disable SUPERSONIC evasion, increase volley evasion instead. For at least if it is overwhelmed, surely this plane should be shut down.
> 
> What do you mean by volley evasion?
>Its the countermeasure. It throws volleys it can be modified how many volley to be fired. Aurora would be vulnerable if overwhelmed.

**Comment by statesman01** [comment]
> > Here's mine. Disable SUPERSONIC evasion, increase volley evasion instead. For at least if it is overwhelmed, surely this plane should be shut down.
> 
> What do you mean by volley evasion?

Its the countermeasure. It throws volleys it can be modified how many volley to be fired. Aurora would be vulnerable if overwhelmed.

**Comment by ZekeDlyoung** [comment]
Countermeasures don't block bullets, meaning a group of Quads/Gatts can kill it easily, making it an overpriced SF with no Bunker Busters. 🤷 

For me I'd rather the Aurora have nothing except its supersonic invulnerability, even if it means reducing its damage

**Comment by ZekeDlyoung** [comment]
Another idea would be to make the Aurora creation voice heard by all players, just like the Kirov in RA2, so other players are at least alerted that someone has built Auroras and act accordingly

**Comment by statesman01** [comment]
Proposal 1 is cool. Increase normal movement speed. disable supersonic evasion. So that the player could strategically utilize the the whole map. Finding weak entry to enemy base is a good strategy rather than simply locking a target. Just buff up Aurora damage so it would be worth it.

**Comment by ExiLe-ZH** [comment]
> I don't think the Aurora invulnerability should be touched, as it is a unique defining characteristic of it.
> 
> That said I have been thinking of how to nerf it in other ways:
> 
> 1. Increase reload time, so other players have more time to kill it while in the airfield/ have more time to recover in between Aurora attacks
> 2. Remove SF GP unlock, and give it to the Aurora instead (I don't think build cost needs to be further increased they're already very expensive)
> 3. Increase the turn radius of the Aurora's slow locomotor so it has a higher chance of getting shot down after it attacks
> 4. Lessen its invincibility miss distance so AA weapons re-target faster after the Aurora comes out of supersonic mode and weapons with large AoE (nuke mig missiles) have a chance of killing them even in supersonic mode
> 5. Alpha Aurora only: Increase delay of their FAB explosion to make it less reliable in hitting moving targets
> 6. Decrease Aurora HP or increase the damage it takes while in low fuel
> 7. All of the above
> 
> If you are set on removing its super-sonic ability, a more interesting alternative to giving it tons of HP is to re-use the lock-on mechanic from ZH. Instead of being invincible while attacking, AA weapons simple have a delay to adjust their aim before firing

I like this thinking a lot more, I like proposal 5 for Alphas too, could nerf the initial damage+radius a bit aswell and keep the blast more powerful but delayed a bit more. Right now large moving groups of quads can be taken out without having to lead the shot, you just target the first one. 


> Countermeasures don't block bullets, meaning a group of Quads/Gatts can kill it easily, making it an overpriced SF with no Bunker Busters. 🤷
> 
> For me I'd rather the Aurora have nothing except its supersonic invulnerability, even if it means reducing its damage

While not described ingame, countermeasures seem to function like a traditional armor boost vs all weapontypes including bullets.

**Comment by ZekeDlyoung** [comment]
> While not described ingame, countermeasures seem to function like a traditional armor boost vs all weapontypes including bullets.

I know. I'm just pointing out that increasing volley evasion as described above does nothing to improve it's effectiveness against hitscan AA, it will just be a big nerf in this case

**Comment by xezon** [comment]
6 proposals added.

**Comment by FDPUser443** [comment]
Proposal 10
1. Aurora is not invincible for Avengers and Laser Turrets (Light Speed > Aurora Speed)

Proposal 11
1. Aurora is not invincible for Avengers and Laser Turrets (Light Speed > Aurora Speed)
2. Aurora is also not invincible for Migs/Raptors/BunkerLix if a rockets fly towards the aurora and angle is not greater than 5-10 degrees, for example;
![image](https://user-images.githubusercontent.com/95628936/191731201-5687ef92-1bb5-4af8-87e4-02918b4c5b3b.png)

**Comment by ExiLe-ZH** [comment]
Very interesting proposals, I like the idea that the Aurora's invincibility is truly a physical ability of dodging treats, not because of invincible armor.

**Comment by xezon** [comment]
Proposal 10 is perhaps possible, Proposal 11 likely not, as we cannot append angle checks in INI, unless there was such a feature already, which is unlikely because in regular game everything damages everything else from any angle.

**Comment by RisingZH** [comment]
I don’t like this one either, would group it in with laser tanks - another change that would make new patch feel less like 1.04 without fixing anything, the whole thing about auroras is they’re invincible until they’ve fired.

On top, non-alpha auroras already suck so why would we want to make a bad unit worse. And whilst alphas are very good, swg is one of the weakest armies so shouldn’t be nerfed.

**Comment by ExiLe-ZH** [comment]
I also agree here with rising, vAuroras are quite useless already indeed.

While I like the concept of the ideas, it all adds extra complexity to gameplay that makes things less predictable. Currently it's simple, supersonic flight dodges everything.

Btw, I believe I already openend a dedicated topic where I made proposals like adjusting vAurora price to 2000 (same as Air and SW). Aswell as reducing experience values/requirements to be 25%? lower than Alphas.

**Comment by xezon** [comment]
If Auroras are so bad, how come they are banned in every non-No Rules match?

**Comment by Stubbjax** [comment]
One word: unfair.

**Comment by ExiLe-ZH** [comment]
Because rules are often played by noobs who are clueless about balance 😄

**Comment by RisingZH** [comment]
Because if your a player who doesn’t want the opponent to cross the half way line before 10 minutes have passed, and even after then only through a very small funnel in the middle of tournament island, the mere concept of an aurora threatens your very existence.

**Comment by xezon** [comment]
Aurora is banned in all rules, including those were Rush is allowed.

**Comment by commy2** [comment]
> Because rules are often played by noobs who are clueless about balance 😄

I said this before, but those rules aren't about balance. They exist to make the game more enjoyable. That's why superweapons are regularly banned: They are less fun than mass armies clashing, which is the whole shtick of C&C (no unit caps). That's also why no-rush-x is a thing: to give beginners more time to build up.

Auroras are a terrible unit design wise, because immortality allows no counter play. They should've never been added to the game, but removing them or redesigning them entirely (like this issue proposes) is out of scope for this patch imho. Since they are an anti fun unit to begin with, I also don't see a point in making them cheaper for vUSA/Laser. This will only upset some people without addressing any balance need for those two factions.

Alphas being OP is a whole different thing, but that is mainly because of their ridiculous damage + splash damage and not (so much) immortality.

**Comment by ExiLe-ZH** [comment]
Ok I can understand the sentiment for such games.

Although I don't see Auroras change campy spam games that much, I guess they can take out Power or Defenses. But I get it anyways.

That said, the definition of fun is subjective and people who play no rules would like to use Aurora's if they were more usefull. It wouldn't hurt anyone to make USA/Laser Auroras a bit more cost efficient, Airforce proves it wouldn't upset anyone and they're banned from rule games anyway.

**Comment by commy2** [comment]
Not hurting the gameplay is a necessary, but not a sufficient condition for introducing a change like this. Would the game be better off with cheaper vAuroras? I don't see it.

**Comment by ExiLe-ZH** [comment]
Yes, currently there's no incentive to make them at all, just make more Vees...meh. Losing them also just hurts too much. 

Seeing them used more (effectively) would just be a beautiful sight, to see problems being solved tactically. Examples:
- Laser taking out long range units, which he normally couldn't fight.
- Or taking out Power/Defenses to break mega bunkered up players.
- Or seeing them take out SWs, which still wouldn't be cost-efficient, 8k vs 5k and giving away 800xp in most cases.

It doesn't matter if it's dirty when it's efficient, that's exactly makes it beautiful.

**Comment by MTKing4** [comment]
Reducing cost of vUSA and Laser auroras to $2000 should be fine
But not to AFG, those ones are stronger, more HP + PDLs

**Comment by ExiLe-ZH** [comment]
AFG's is already 2000 :D

**Comment by MTKing4** [comment]
Nvm then 😄

**Comment by ExiLe-ZH** [comment]
My stance hasn't changed on high profile changes, but I was just brainstorming a little, like an alternative could be to restrict the amount of time it can go super sonic. Imagine this being a case of limited fuel capacity, or combat radius as it's called in the real world. This way you might have to micro around danger zones manually before locking onto target.

**Comment by MTKing4** [comment]
Or maybe it can only go super sonic before X distance from target

**Comment by xezon** [comment]
If I am not mistaken the invincibility aspect of the Aurora can be exploited over enemy base defenses. Send your main ground forces with Attack Move toward the enemy base defenses, then take 1 or 2 Auroras and Force Attack repeatedly on the opposite sides of the enemy defenses so that it continuesly flies over it (Click on Radar for Attack to make it easier to do). Then enemy Anti Air defenses (Palace, Stinger, Turrets, Rockets) will shoot on Aurora. but Aurora will never die because it is in Hypersonic mode. Enemy player needs to try target enemy ground forces manually, so Aurora effect can be mitigated. Still, this trick or exploit will likely upset the opposition, because there is no counterplay to it, meaning Aurora cannot be killed.

Also. With Hypersonic Aurora you can scout enemy base at all times. Just Force Attack through enemy base, then return back home with Force Attack too. Aurora cannot be prevented from inspecting base. It's not good.

**Comment by MTKing4** [comment]
> If I am not mistaken the invincibility aspect of the Aurora can be exploited over enemy base defenses. Send your main ground forces with Attack Move toward the enemy base defenses, then take 1 or 2 Auroras and Force Attack repeatedly on the opposite sides of the enemy defenses so that it continuesly flies over it (Click on Radar for Attack to make it easier to do). Then enemy Anti Air defenses (Palace, Stinger, Turrets, Rockets) will shoot on Aurora. but Aurora will never die because it is in Hypersonic mode. Enemy player needs to try target enemy ground forces manually, so Aurora effect can be mitigated. Still, this trick or exploit will likely upset the opposition, because there is no counterplay to it, meaning Aurora cannot be killed

I have never seen anyone do that, nor do i think is possible at all, you can't continue being in supersonic speed when you're turning, that's just not possible, Aurora is vulnerable then, you can't go back and forth on enemy defenses to distract them, eventually you'll have to turn back, and you will be vulnerable.

**Comment by commy2** [comment]
> nor do i think is possible at all, you can't continue being in supersonic speed when you're turning, that's just not possible, Aurora is vulnerable then, you can't go back and forth on enemy defenses to distract them, eventually you'll have to turn back, and you will be vulnerable.

https://github.com/TheSuperHackers/GeneralsGamePatch/assets/6576312/df3255b9-05a2-4761-afad-fa1ba3cea78a

**Comment by xezon** [comment]
Thank you for checking.

Why no players abuse this I do not know. Perhaps because doing this requires too much work? In any case, if Aurora would take a bit of damage in Hypersonic, then these shenanigans would no longer be possible.

**Comment by ExiLe-ZH** [comment]
Yes it's possible as I already knew, but the reason nobody uses it is because it's very micro intensive, you barely have time to micro your actual army when doing this. Neither is it possible to force fire on the minimap (quite positive about that), so you have to keep scrolling around. Anyone who uses this succesfully deserves a little applause imo.

**Comment by MTKing4** [comment]
> > nor do i think is possible at all, you can't continue being in supersonic speed when you're turning, that's just not possible, Aurora is vulnerable then, you can't go back and forth on enemy defenses to distract them, eventually you'll have to turn back, and you will be vulnerable.
> 
>  Untitled.mp4

That's not how i imagined it to work at all 👀

I guess the reason i thought it can't dodge while turning is the fact that most people who i've seen do this usually end up mismicroing by delaying the next movement, which causes the plane to leave supersonic stance and then die. Because, as Exile said, it's quite micro intensive, you have to keep forcefiring, while also ensuring that it won't drop the bomb before issuing the next command, which is a small time window, so there's a lot of timing and care put to it to be achieved, which makes it not very useful in actual games.
